Update parameters for legacy Exchange tokens (#12360)
MicrosoftDocs/office-docs-powershell: exchange/exchange-ps/exchange/Remove-AuthenticationPolicy.md

This example removes the authentication policy named "Engineering Group".

### Example 2
```powershell

> Remove-AuthenticationPolicy -Identity "Legacy Exchange Tokens" -AllowLegacyExchangeTokens

Reviewer: ```suggestion
Remove-AuthenticationPolicy -Identity "LegacyExchangeTokens" -AllowLegacyExchangeTokens
```

```

> In Exchange Online, this example returns your organization to its previous state before legacy Exchange token issuance was allowed or blocked for Outlook add-ins. Since this switch applies to the entire organization, the authentication policy specified with the Identity parameter is ignored.

Reviewer: ```suggestion
In Exchange Online, this example enables legacy Exchange tokens to be issued to Outlook add-ins. This switch applies to the entire organization. The Identity parameter is required, and its value must be set to "LegacyExchangeTokens". Specific authentication polices can't be applied.
```

### -AllowLegacyExchangeTokens
This parameter is available only in the cloud-based service.

> The AllowLegacyExchangeTokens switch returns your organization to its previous state before changes were made to allow or block legacy Exchange tokens for Outlook add-ins. You don't need to specify a value with this switch.

Reviewer: ```suggestion
The AllowLegacyExchangeTokens switch enables legacy Exchange tokens to be issued to Outlook add-ins for your organization. You don't need to specify a value with this switch.
```

Legacy Exchange tokens include Exchange user identity and callback tokens.

> This switch applies to the entire organization. Although the Identity parameter is required, its value is ignored. You can pass any non-empty value as the Identity parameter.

Reviewer: ```suggestion
This switch applies to the entire organization. The Identity parameter is required, and its value must be set to "LegacyExchangeTokens". Specific authentication polices can't be applied.
```
